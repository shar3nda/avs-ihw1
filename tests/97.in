10
6 3 5 -2 7 3 3 10 1 -1