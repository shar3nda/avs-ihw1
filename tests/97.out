-2 -3 -2 7 -8 -10 -1 1 -6 0